src/lib/utils/optimise-cloudinary.ts

```
@@ -0,0 +1,66 @@
+        baseUrl = url.substring(0, uploadIndex + '/upload'.length);
+        path = url.substring(uploadIndex + '/upload/'.length);
+    } else if (coverImageIndex !== -1) {
+        baseUrl = url.substring(0, coverImageIndex);
+        path = url.substring(coverImageIndex + '/coverImage/'.length);
+    } else {
+        return url;
+    }
+
+    if (!path) return url;
+
+    let transformParams: string[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'transformParams' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)